src/ClassyResource/main.js

```diff
@@ -24,6 +24,11 @@ export default class ClassyResource {
       });
     }
 
+    /** Set up Error Logger (if defined) */
+    if (this._classy.errorLogger) {
```

Comment:
why is this needed?

Reply:
Honestly, the ClassyResource vs. Classy convention in Classy-Node is odd and I don't really understand the intent. I did it this way to maintain the existing pattern. It seems like the intent originally was to define Classy defaults using the Classy content and use the ClassyResource to override those defaults if need be.